app/views/shared/_help_icon.html.erb

```diff
@@ -0,0 +1,3 @@
+<a class="admin-help-button cursor-pointer p-2 mb-2 z-elevate m:p-0 m:pr-3 crayons-btn crayons-btn--icon-rounded crayons-btn--s" href="<%= @help_url %>" target="_blank" rel="noopener noreferer">
+  <%= inline_svg_tag("circle-question.svg", aria: true, title: "Forem Admin Guide") %>
+</a>
```

Comment:
```suggestion
<a class="admin-help-button fixed bottom-7 right-3 z-elevate cursor-pointer radius-full l:right-7 crayons-btn crayons-btn--icon-rounded crayons-btn--s" href="<%= @help_url %>" target="_blank" rel="noopener noreferer">
  <%= inline_svg_tag("circle-question.svg", aria: true, class: "block", title: "Forem Admin Guide") %>
</a>
```

@juliannatetreault here's an update of the classes. The reason that it was not displaying as per the admin interface is because there were a few properties missing, namely the ones that are responsible for the positioning of the help button.

This was the initial css for the admin help button:
```
.admin-help-button {
  position: fixed;
  bottom: var(--su-7);
  margin-bottom: env(safe-area-inset-bottom);
  right: var(--su-7);
  z-index: var(--z-elevate);
  cursor: pointer;
  border-radius: 50px;

  @media (max-width: $breakpoint-m) {
    bottom: calc(var(--header-height) + var(--su-3));
    right: var(--su-3);
    padding: 0;
  }
}
```

Let's map for each css attribute to a utility class

| CSS Attribute      | Utility Class | Description |
| ----------- | ----------- | -------------- |
| position: fixed      | fixed       | This was missing and forces the icon to be fixed in a certain place on the screen |
| bottom: var(--su-7)   | bottom-7        | This was missing. We want the icon to be fixed 2rem (su--7) from the bottom |
| margin-bottom: env(safe-area-inset-bottom)   | No utility class       | You can read more about it [here](https://developer.mozilla.org/en-US/docs/Web/CSS/env())|
| right: var(--su-7);   | right-7    | This was missing. We want the icon to be fixed 2rem (su--7) to the right of the right edge of the screen |
| z-index: var(--z-elevate);   | z-elevate        | elevates the icon to the top |
| cursor: pointer;   | cursor-pointer        | pointer cursor |
| border-radius: 50px;   | radius-full        | Adds rounded edges, I explain the border radius value a little more in a different comment, but essentially we just want to border to allow the button shape to be circular, hence any high value would work |
| Media Queries  for medium screens|   |
| bottom: calc(var(--header-height) + var(--su-3));   | No utility class        | There is calculation here so it has to stay in the css stylesheets |
| right: var(--su-3);   | right-3        | For a medium screen or smaller we want the icon to be fixed 0.75rem (su--3) to the right of the right edge of the screen. Also note that `pr-3` is `padding-right: var(--su-3)` and not the positional element `right`.  |
|padding: 0;  | p-0        | On medium and below screens there must be no padding. I've removed this attribute because we are not setting padding on this item so there was no need to override it to be 0 in mobile. |

Using this table you can recognize the ones that were missing. I've also updated the css to align with using a `suggestion` box.

One other thing to note here, is that the way that utility classes behave for responsive design is as follows: I've taken an excerpt from the [tailwind docs](https://tailwindcss.com/docs/responsive-design), even though we're not using tailwind, we still apply the same principles.

```
Mobile First
By default, Tailwind uses a mobile first breakpoint system, similar to what you might be used to in other frameworks like Bootstrap.

What this means is that unprefixed utilities (like uppercase) take effect on all screen sizes, while prefixed utilities (like md:uppercase) only take effect at the specified breakpoint and above.

Targeting mobile screens
Where this approach surprises people most often is that to style something for mobile, you need to use the unprefixed version of a utility, not the sm: prefixed version. Don’t think of sm: as meaning “on small screens”, think of it as “at the small breakpoint“.
```

Hence you will notice that the right utility class `right-3` is applied to all screens that are smaller than large, and we prefix `right-7` with `l` which is indicates a large screen.

This can all be overwhelming, so if anything is unclear, please feel free to ping me. 😊

Reply:
Updated placement and styles:

<img width="846" alt="Screenshot 2021-11-11 at 12 13 05" src="https://user-images.githubusercontent.com/2786819/141280292-55c8de11-3925-4829-8724-feb0e1254cc5.png">

Reply:
This was all so incredibly useful! Thank you so much for walking me through the why--I sincerely appreciate it and learned so much. 🙌 I've made these changes per your suggestions.